### PR TITLE
FAT-5616 Created list of products for user and admin

### PR DESCRIPTION
**Pull request by Activict**

### CODE REVIEW

File: StoreApp/Controllers/AccountController.cs [hunk 1]
```diff
@@ -123,6 +134,26 @@ public ActionResult Account()
...
+            foreach (var order in orders)
+            {
+                order.OrderDetails = webMapper.config.Map<IEnumerable<OrderDetailDTO>, IEnumerable<OrderDetailsViewModel>>(orderDetailService.GetAll().Where(o => o.OrderId == order.Id));
+
+                foreach (var details in order.OrderDetails)
+                {
+                    details.Product = webMapper.config.Map<ProductDTO, ProductViewModel>(productService.Get(details.ProductId));
+                    details.Product.Unit = unitService.Get(details.Product.UnitId).Name;
+                }
+            }
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/AccountController.cs:152]: Try to optimize

File: StoreApp/Views/Account/UserOrders.cshtml [hunk 2]
```diff
@@ -0,0 +1,91 @@
+﻿@model IEnumerable<StoreApp.Models.Orders.OrderViewModel>
+
+@{
+    ViewBag.Title = User.Identity.Name + " orders";
+}
+
+<h3>@User.Identity.Name orders</h3>
+
+
```
Comment (AleksandroBoko) [inline StoreApp/Views/Account/UserOrders.cshtml]: Remove empty space

File: StoreApp/Views/Account/UserOrders.cshtml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+        <div class="col-1">
+            @Html.DisplayFor(modelItem => item.Status)
+        </div>
+        <div class="col-1">
+            @Html.DisplayFor(modelItem => item.Discount)
+        </div>
+        <div class="col-1">
+
+        </div>
```
Comment (AleksandroBoko) [inline StoreApp/Views/Account/UserOrders.cshtml]: What is it?

File: StoreApp/Views/Account/UserOrders.cshtml (hunk 2 above)
Comment (Activict) [reply StoreApp/Views/Account/UserOrders.cshtml]: fixed

File: StoreApp/Views/Account/UserOrders.cshtml (hunk 3 above)
Comment (Activict) [reply StoreApp/Views/Account/UserOrders.cshtml]: fixed

**Pull request merged**